docs/user-guide/how-to-deploy.rst

```diff
@@ -1,21 +1,64 @@
-:desc: Run and ship your Rasa assistant with Docker containers on any
-       Docker-compatible machine or cluster.
+:desc: How to deploy your Rasa Assistant with Docker Compose or Kubernetes/Openshift
 
-.. _running-rasa-with-docker:
+.. _deploying-your-rasa-assistant:
 
-Running Rasa with Docker
-========================
+Deploying your Rasa Assistant
+===============================
 
 .. edit-link::
 
-This is a guide on how to build a Rasa assistant with Docker.
-If you haven't used Rasa before, we'd recommend that you start with the :ref:`rasa-tutorial`.
+This page explains when and how to deploy an assistant built with Rasa. 
+It will allow you to make your assistant available to users and set you up with a production-ready environment.
 
 .. contents::
    :local:
+   :depth: 2
+
+
+When to deploy your assistant
+--------------------------------
+
+.. raw:: html
+
+    The best time to deploy your assistant and make it available to test users is once it can handle the most important happy paths or is what we call a <a style="text-decoration: none" href="https://rasa.com/docs/rasa/glossary">minimum viable assistant</a>.
+
+The recommended deployment methods described below make it easy to share your assistant with test users via the `share your assistant feature in Rasa X <../../rasa-x/docs/user-guide/enable-workflows#conversations-with-test-users>`_. Then, when you’re ready to make your assistant available via one or more :ref:`messaging-and-voice-channels`, you can easily add them to your existing deployment set up.
+
+.. _recommended-deployment-methods:
+
+Recommended Deployment Methods
+------------------------------
+
+The recommended way to deploy an assistant is using either the Docker Compose or Kubernetes/Openshift options we support. Both deploy Rasa X and your assistant. They are the easiest ways to deploy your assistant, allow you to use Rasa X to view conversations and turn them into training data, and are production-ready.
+
+Kubernetes/Openshift
+~~~~~~~~~~~~~~~~~~~~
+
+Kubernetes/Openshift is the best option if you plan to serve many users. It's straightforward to deploy if you use the helm charts we provide. However, you can also customize the Helm charts if you have specific requirements.
+
+    - Default: Read the docs `here <../../rasa-x/docs/installation-and-setup/openshift-kubernetes/>`_.
```

Comment:
you also need to add two underscores at the end of the link, otherwise the targets won't render properly

Reply:
for each "here" reference: https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557

Reply:
added